repository-data/webfiles/src/main/resources/site/freemarker/common/static-ui/component/hub-box.ftl

```diff
@@ -0,0 +1,97 @@
+<#ftl output_format="HTML">
+<#include "../../macro/component/icon.ftl">
+<#include "../../macro/cyberAlertBox.ftl">
+<#include "../../macro/hubBox.ftl">
+<#include "../../../include/imports.ftl">
+
+
+<#assign cyberAlertData = {
+    "title": "Title",
+    "text": "Proactively enhance high standards in deliverables.",
+    "severity": "critical",
+    "threatType": "critical",
+    "threatId": "Worm",
+    "types": ["Type A", "Type B"],
+    "background": "/site/webfiles/1580485506348/images/fibre_57101102_med.jpg",
```

Comment:
This URL will change every time the website reboots. As it is an FTL file, I think you need `<@hst.webfile  path="images/fibre_57101102_med.jpg" />`.

Reply:
This is just throw-away code, so it doesn't matter if the static file name changes.

Reply:
If it needs committing, then it should be committed in a working state.

Reply:
Good point, I'll add the webfile link.

Reply:
It turns out we didn't actually need the image for the static ui, so @josephcyrus-valtech removed it.

Reply:
Okay